README.md

```diff
@@ -92,13 +92,65 @@ all:
 
 This playbook requires a user to set the following set of variables:
 
-- **maas_version**: The intended version of MAAS to install. Example: '3.2'
+- **maas_version**: The intended version of MAAS to install.\
+  Example: `'3.2'`
 
-- **maas_postgres_password**: The password for the MAAS postgres user. Example: 'my_password'
+- **maas_postgres_password**: The password for the MAAS postgres user.\
+  Example: `'my_password'`
 
-- **maas_installation_type**: The intended type of MAAS installation. Possible values are: 'deb' or 'snap'
+- **maas_installation_type**: The intended type of MAAS installation.\
+  Possible values are: `'deb'` or `'snap'`
 
-- **maas_url**: The MAAS URL where MAAS will be accessed and which rack controllers should use for registering with region controllers. Example: 'http://proxy01.example.com:5240/MAAS'
+
+- **maas_url**: The MAAS URL where MAAS will be accessed and which rack controllers should use for registering with region controllers.\
+  Example: `'http://proxy01.example.com:5240/MAAS'`
+
+
+There are additional optional variables that can be passed to the playbooks:
+
+- [Admin credentials] [If setting up an admin user]
```

Comment:
Not sure, I've been going back and forth on how to present that some parameter apply to a specific thing (ie, these are grouped because they are for the admin user)